### PR TITLE
Fixed some typos in Mastering Plone 6 > 4, 11, 12

### PR DESCRIPTION
**Pull request by curiousyuvi**

### CODE REVIEW

File: docs/mastering-plone/case.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ For this training we will build a website for a fictional Plone conference on Ma
 
 By 2050 human civilisation has reached the stars and each year the conference is held on a different planet.
 
-The conference-webseite we want to create should be reusable in the following years so some things should be configurable.
+The conference-website we want to create should be reusable in the following years so some things should be configurable.
```
Comment (stevepiercy) [inline docs/mastering-plone/case.md:22]: You might see an overabundance of hyphenated words, such as this. Germanic languages do this a lot, but in English it is less common.

```suggestion
The conference website we want to create should be reusable in the following years, so some things should be configurable.
```

File: docs/mastering-plone/case.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ During the course of the training you will solve the following tasks.
 - Create a sponsor contenttype to manage sponsors
 - Store non-visible information about the sponsor in the sponsor-type
 - Display sponsors on all pages sorted by level
-- Allow some avilable to be configurable by a admin in a controlpanel
+- Allow some available to be configurable by a admin in a controlpanel
```
Comment (stevepiercy) [inline docs/mastering-plone/case.md:81]: This sentence does not make sense. It is missing a word after "available", or perhaps the intent is "availability"? I am not sure. Would you please look elsewhere in this Training to infer what it should be?

```suggestion
- Allow some available to be configurable by an admin in a control panel
```

File: docs/mastering-plone/volto_overrides.md [hunk 3]
```diff
@@ -313,7 +313,7 @@ So we'll add some simple logic to use the effective date only if it exists.
 )}
 ```
 
-As we are in the html part of our Rect component, we embrace the Javascript code in curly braces.
+As we are in the html part of our React component, we embrace the Javascript code in curly braces.
```
Comment (stevepiercy) [inline docs/mastering-plone/volto_overrides.md:316]:
```suggestion
As we are in the HTML part of our React component, we surround the JavaScript code with curly braces.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
English grammar fixes.

**Comment by stevepiercy** [comment]
@ksuess @pbauer do either of you know what is the intent of this sentence? It's missing a word.

https://github.com/plone/training/pull/733/files#diff-dd0fff211cdaf08da26e0acdce999187e533ccf36f118d9605d825471169fdabR81

File: docs/mastering-plone/case.md [hunk 4]
```diff
@@ -78,7 +78,7 @@ During the course of the training you will solve the following tasks.
 - Create a sponsor contenttype to manage sponsors
 - Store non-visible information about the sponsor in the sponsor-type
 - Display sponsors on all pages sorted by level
-- Allow some avilable to be configurable by a admin in a controlpanel
+- Allow some available to be configurable by an admin in a control panel
```
Comment (ksuess) [inline docs/mastering-plone/case.md:81]:
```suggestion
- Allow some settings to be configurable by an admin in a control panel
```

**Review by ksuess — CHANGES_REQUESTED** [review]

**Review by stevepiercy — APPROVED** [review]
Thank you!

**Pull request merged**